syncano/models/manager.py

```diff
@@ -828,6 +828,26 @@ def _get_endpoint_properties(self):
         return self.model._meta.resolve_endpoint(self.endpoint, defaults), defaults
 
 
+class RuntimeChoices(object):
```

Comment:
Yep - thinkin bout that :) That user must do some inspection.

How do you want to provide constants, if runtime_names can change? dynamically? 

Do you have any idea?